broker-node/app/ChunkEvents.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+
+namespace App;
+
+
+
+use App\Clients\BrokerNode;
+
+use Illuminate\Database\Eloquent\Model;
+
+use Webpatser\Uuid\Uuid;
+
+
+
+class ChunkEvents extends Model
```

Comment:
Nit: Models are typically singular (ChunkEvent) while tables and controllers are plural (chunk_events, ChunkEventsController)

Reply:
k
I will look through everything and conform to standards.  Wouldn't mind a standards guide.